vmdb/lib/miq_automation_engine/service_models/miq_ae_service_miq_provision_task.rb

```diff
@@ -0,0 +1,13 @@
+module MiqAeMethodService
+  class MiqAeServiceMiqProvisionTask < MiqAeServiceMiqRequestTask
+    def statemachine_task_status
```

Comment:
This is only used by the automate state machine to loop during the provisioning state.  I like it here as a helper method to the automate methods since the backend models have no need for it.